source/common/orca/orca_parser.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "envoy/http/header_map.h"
+
+#include "absl/status/statusor.h"
+#include "xds/data/orca/v3/orca_load_report.pb.h"
+
+namespace Envoy {
+namespace Orca {
+
+// Headers used to send ORCA load metrics from the backend.
+static constexpr char kEndpointLoadMetricsHeader[] = "endpoint-load-metrics";
+static constexpr char kEndpointLoadMetricsHeaderBin[] = "endpoint-load-metrics-bin";
+static constexpr char kEndpointLoadMetricsHeaderJson[] = "endpoint-load-metrics-json";
```

Comment:
Is there a doc about the header name and format definition? And `constexpr absl::string_view` is prefered.

Reply:
/wait-any

Reply:
[The public de](https://docs.google.com/document/d/1gb_2pcNnEzTgo1EJ6w1Ol7O-EH-O_Ysu5o215N9MTAg/edit#heading=h.bi4e79pb39fe)

Reply:
The above doc is Envoy's implementation and may be changed when we reviewing it

If we want the ORCA to be open standard, then I think the header format should at least be part of https://docs.google.com/document/d/1NSnK3346BkBo1JUU3I9I5NYYnaJZQPt8_Z_XCBCI3uA/edit#heading=h.xgjl2srtytjt

If it's not specified, how the gRPC proxy implement it? We may can also take the gRPC as a reference.

Reply:
Resolving this as these questions have been addressed in the other comment thread.